addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1048,7 +1048,10 @@ msgctxt "#225"
 msgid "Vertical shift"
 msgstr ""
 
-#empty string with id 226
+#: system/settings/settings.xml
+msgctxt "#226"
```

Comment:
In strings.po the use (reuse probably) of low id is to be avoided, they are impossible to search for in the code. Instead add a new string at the end of the file.